src/components/Timer/index.tsx

```diff
@@ -21,18 +22,27 @@ export default function Question({
   selectedQuestionId,
 }: Props) {
   const [remainingTime, setRemainingTime] = useState(duration);
+  const [circleDasharray, setCircleDasharray] = useState('283');
```

Comment:
Why 283?

Reply:
it is circumference of timer circle `(2 * pi * r) where r  = 45`.
read this for better understanding
https://css-tricks.com/how-to-create-an-animated-countdown-timer-with-html-css-and-javascript/

Reply:
Can you type create a variable timerCircleCircumference = (2 * pi * 45) and then use it?